$NetBSD: patch-configure.ac,v 1.1 2012/12/15 09:36:21 marino Exp $

* Disable -Werror for these reasons
  - Numerous g_thread_init deprecated errors
  - Numerous set-but-not-used error (gcc 4.6+)

--- configure.ac.orig	2008-12-09 08:22:40.000000000 +0000
+++ configure.ac
@@ -27,7 +27,7 @@ dnl ------------------------------------
 dnl - Extra verbose warning switches
 dnl ---------------------------------------------------------------------------
 if test "$GCC" = "yes"; then
-	CPPFLAGS="$CPPFLAGS -Werror -Wcast-align -Wno-uninitialized"
+	CPPFLAGS="$CPPFLAGS -Wcast-align -Wno-uninitialized"
 	CPPFLAGS="$CPPFLAGS -Wall -Wformat-security"
 #	CPPFLAGS="$CPPFLAGS -DG_DISABLE_SINGLE_INCLUDES -DGTK_DISABLE_SINGLE_INCLUDES"
 	CPPFLAGS="$CPPFLAGS -DG_DISABLE_DEPRECATED -DGTK_DISABLE_DEPRECATED -DGDK_DISABLE_DEPRECATED"
